src/ffi/ffiexports.cpp

```diff
@@ -208,6 +209,16 @@ int getHighestBlock() {
                             : (int) ::ChainActive().Height(); // return current block count if no peers
 }
 
+// Returns Major, Minor, Revision
+std::array<uint64_t, 3> getClientVersion() {
```

Comment:
Couldn't we use existing `std::string FormatVersion(int nVersion)` here ?

Reply:
woops, didn't see this XD